OpenRA.Mods.Common/Widgets/SpriteWidget.cs

```diff
@@ -61,7 +63,7 @@ public override void Draw()
 
 			if (sprite != cachedSprite)
```

Comment:
This should probably get a `|| scale != cachedScale` and be combined with the if below to reduce duplication.